common/src/test/java/gov/cms/ab2d/common/util/AB2DLocalstackContainer.java

```diff
@@ -0,0 +1,27 @@
+package gov.cms.ab2d.common.util;
+
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.localstack.LocalStackContainer;
+import org.testcontainers.utility.DockerImageName;
+
+
+@Slf4j
+public class AB2DLocalstackContainer extends LocalStackContainer {
```

Comment:
Were you not able to use this version in other modules? This class is copied multiple times except it's renamed.

Reply:
no, because its a test class, those don't get applied when using dependents. In the future we might create a future client with all default classes like this.

Reply:
Oh yeah I see now. API and worker use spring to pull in the common version. The other modules need their own.